CONTRIBUTING.md

```diff
@@ -87,7 +87,8 @@ git clone https://github.com/pytorch/ignite.git
 cd ignite
 python setup.py develop
 pip install -r requirements-dev.txt
-pip install flake8 "black==19.10b0" "isort==5.7.0" mypy
+pip install mypy
```

Comment:
shouldn't we install mypy with `run_code_style.sh install` ?

Reply:
It will install mypy for mac and windows in the CI.
If its okay for you, we can add mypy in the script.

Reply:
I think it is OK as far as we do not execute it.